src/src/common.h

```diff
@@ -73,14 +73,14 @@ typedef enum
 
 typedef struct expresslrs_rf_pref_params_s
 {
-    int8_t index;
-    expresslrs_RFrates_e enum_rate; // Max value of 16 since only 4 bits have been assigned in the sync package.
-    int32_t RXsensitivity;          // expected RF sensitivity based on
-    uint32_t TOA;                   // time on air in microseconds
-    uint32_t RFmodeCycleInterval;
-    uint32_t RFmodeCycleAddtionalTime;
-    uint32_t SyncPktIntervalDisconnected; // how often to send the SYNC_PACKET packet (ms) when there is no response from RX
-    uint32_t SyncPktIntervalConnected; // how often to send the SYNC_PACKET packet (ms) when there we have a connection
+    const int8_t index;
```

Comment:
no need to add `const` to definitions. To be removed.

Reply:
Does it make no difference to optimisation?

Reply:
Yes but no point to add it into every variable. Just define variable to const when used.
These params are used a lot so better to check memory location carefully. This should be in RAM, imo.

Reply:
so in the array most params are const but some change (eg tlm ratio) what's the best way to structure that?

Reply:
Maybe it is better to remove consts and leave this now.
That way it is placed into RAM for sure and is fast. 
Const makes sense here but need to be checked from asm and map to see how the data is used.